lib/matplotlib/gridspec.py

```diff
@@ -350,8 +356,13 @@ def update(self, **kwargs):
 
     def get_subplot_params(self, figure=None):
         """
-        Return a dictionary of subplot layout parameters. The default
-        parameters are from rcParams unless a figure attribute is set.
+        Return the `~.SubplotParams` for the GridSpec.
+
+        In order of preference the values are taken from
+
+        - attributes of the GridSpec
+        - the provided *figure*
+        - :rc:`figure.subplot.*`
```

Comment:
In order of *whose* preference?

Isn't the order rather (1) figure, (2) rcparams, (3) gridspec (that's how I read the code)?

Reply:
Ah, no, there isn't really any preference. It's just that very order. So maybe pre*c*edence instead of *pre*f*erence?